api/db.py

```diff
@@ -6,6 +6,7 @@
 
 """Database abstraction"""
 
+from fastapi_pagination.ext.motor import paginate
```

Comment:
Maybe keep the imports in alphabetical order?  It's not a strict requirement nor a `pylint` check but it helps avoid duplicating lines or finding existing imports when the list grows in general.

Reply:
Yes, I updated the import order.